core/storage/user/gae_models.py

```diff
@@ -2242,9 +2246,6 @@ class UserAuthDetailsModel(base_models.BaseModel):
     # Authentication detail for sign-in using google id (GAE). Exists only
     # for full users. None for profile users.
     gae_id = ndb.StringProperty(indexed=True)
-    # A code associated with profile and full user on Android to provide a PIN
-    # based authentication within the account.
-    pin = ndb.StringProperty(default=None)
```

Comment:
Do we need to remove this value from the existing models?

Reply:
Currently the value of this attribute is **None** for all users since there does not exist a way as of now to populate this field. So we have to drop the attribute but not exactly remove "values".